src/Sylius/Bundle/CoreBundle/Resources/config/app/workflow/sylius_order_shipping.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+use Sylius\Component\Core\Model\Order;
+use Symfony\Config\FrameworkConfig;
+
+return static function (FrameworkConfig $framework): void {
+    $orderShipping = $framework->workflows()->workflows('sylius_order_shipping');
+    $orderShipping
+        ->type('state_machine')
+        ->supports([Order::class])
+        ->initialMarking(['cart']);
+
+    $orderShipping->markingStore()
+        ->type('method')
+        ->property('shippingState');
+
+    $orderShipping->place()->name(Sylius\Component\Core\OrderShippingStates::STATE_CART);
```

Comment:
I'm not familiar with php configs, but is to possible to add `use` for OrderShippingStates?

```suggestion
    $orderShipping->place()->name(OrderShippingStates::STATE_CART);
```